.github/workflows/mt-annotations.yaml

```diff
@@ -54,10 +54,18 @@ jobs:
               run: |
                   composer install --no-interaction --prefer-dist --no-progress
 
-            - name: Run Infection for added files only
+            - name: Run Infection for changed lines only
               run: |
                   git fetch origin $GITHUB_BASE_REF
-                  php bin/infection --threads=max --git-diff-lines --git-diff-base=origin/$GITHUB_BASE_REF --ignore-msi-with-no-mutations --only-covered
+                  php bin/infection \
+                    --threads=max \
+                    --git-diff-lines \
+                    --git-diff-base=origin/$GITHUB_BASE_REF \
+                    --ignore-msi-with-no-mutations \
+                    --min-msi=100 \
+                    --min-covered-msi=100 \
+                    --static-analysis-tool=phpstan \
```

Comment:
```suggestion
```
Can we discuss this separately?

Reply:
I'm ok with extracting it to a separate discussion